### PR TITLE
Move Push Taskqueue sample program to Github

### PR DESCRIPTION
**Pull request by Annie29**
@lesv, @tswast 
Moves files into Github for Java Taskqueue example. 

Some minor corrections to Requests testing and configuration.  This should replace [Pull Request 205](https://github.com/GoogleCloudPlatform/java-docs-samples/pull/205).

### CODE REVIEW

File: appengine/taskqueue-push/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Go to the site `localhost:8080` to add elements to the queue.  They will appear in the log as the result of the Enqueue servlet transmitting the data to the Worker servlet.
+
+## Deploying
+In the following command, replace YOUR-PROJECT-ID with your
+[Google Cloud Project ID](https://support.google.com/cloud/answer/6158840).
+
+    $ mvn appengine:update -Dappengine.appId=YOUR-PROJECT-ID -Dappengine.version=SOME-VERSION
+
+## Setup
```
Comment (lesv) [inline appengine/taskqueue-push/README.md]: Redundant

File: appengine/taskqueue-push/src/main/java/com/example/appengine/taskqueue/push/Worker.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+//  [START worker]
+// The Worker servlet should be mapped to the "/worker" URL.
+public class Worker extends HttpServlet {
+  private static final Logger log = Logger.getLogger(Worker.class.getName());
+
+  protected void doPost(HttpServletRequest request, HttpServletResponse response)
+          throws ServletException, IOException {
+    String key = request.getParameter("key");
+
+    // Do something with key.
```
Comment (lesv) [inline appengine/taskqueue-push/src/main/java/com/example/appengine/taskqueue/push/Worker.java]: helpful

File: appengine/taskqueue-push/src/main/webapp/WEB-INF/web.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://java.sun.com/xml/ns/javaee"
```
Comment (lesv) [inline appengine/taskqueue-push/src/main/webapp/WEB-INF/web.xml]: Licence?

File: appengine/taskqueue-push/src/main/webapp/WEB-INF/appengine-web.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (lesv) [inline appengine/taskqueue-push/src/main/webapp/WEB-INF/appengine-web.xml]: License?

File: appengine/taskqueue-push/src/main/webapp/index.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<!--[START index_file] -->
```
Comment (lesv) [inline appengine/taskqueue-push/src/main/webapp/index.html]: License, `<!DOCTYPE html>` ??

File: appengine/taskqueue-push/src/test/java/com/example/appengine/taskqueue/push/WorkerTest.java [hunk 6]
```diff
@@ -0,0 +1,65 @@
+package com.example.appengine.taskqueue.push;
```
Comment (lesv) [inline appengine/taskqueue-push/src/test/java/com/example/appengine/taskqueue/push/WorkerTest.java]: License?

**Comment by lesv** [comment]
PTAL

File: appengine/taskqueue-push/README.md (hunk 1 above)
Comment (Annie29) [inline appengine/taskqueue-push/README.md]: I really don't think so.

Okay, @lesv, now that I see the whole file, I'm just echoing the form @tswast uses.

- Running locally (obvious)
- Deploying (obvious, but this is the form used where the user specifies the project id and version on the command line
- Setup, which is deploying, but putting the parameters in a config uses.

File: appengine/taskqueue-push/README.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+To save your project settings so that you don't need to enter the
+ parameters, you can:
+
+1. Update the `<application>` tag in src/main/webapp/WEB-INF/appengine-web.xml
```
Comment (lesv) [inline appengine/taskqueue-push/README.md]: This isn't required if you do line 22

File: appengine/taskqueue-push/README.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+   with your project name.
+
+2. Update the `<version>` tag in src/main/webapp/WEB-INF/appengine-web.xml
```
Comment (lesv) [inline appengine/taskqueue-push/README.md]: Nor is this.

File: appengine/taskqueue-push/README.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+   with a valid version number.
+
+
+You will now be able to run
+
+    $ mvn appengine:update
```
Comment (lesv) [inline appengine/taskqueue-push/README.md]: And this is also on line 22.

**Comment by lesv** [comment]
PTAL

**Comment by Annie29** [comment]
@lesv I've added lots of licenses.  I haven't changed the README.md.  See example from @tswast above.  If you really want this one changed, we'll need to change a lot of them.

**Comment by lesv** [comment]
No Problem

**Comment by lesv** [comment]
LGTM

**Pull request merged**